iothub/service/src/IotHubConnection.cs

```diff
@@ -553,7 +553,7 @@ private async Task SendCbsTokenAsync(AmqpCbsLink cbsLink, TimeSpan timeout)
                     Credential.AmqpEndpoint,
                     audience,
                     resource,
-                    AccessRightsHelper.AccessRightsToStringArray(_accessRights),
+                    Credential.AmqpAudience,
```

Comment:
The IoT hub service ignores this value for AAD auth. But for the sake of correctness, we should not be passing audience related to SAS tokens during AAD auth.